packages/amplication-git-pull-request-service/src/core/commit/commit.module.ts

```diff
@@ -0,0 +1,9 @@
+import { Module } from '@nestjs/common';
+import { CommitController } from './commit.controller';
+
+@Module({
+  controllers: [CommitController],
+  imports: [CommitController],
+  providers: [CommitController],
```

Comment:
```suggestion
  controllers: [CommitController],
```

Adding `CommitController` to `imports` and `providers` won't do anything.

`imports` expects modules, `providers` expects `@Injectable` classes.